for-import/temp-buttons.css

```diff
@@ -0,0 +1,93 @@
+@import"../for-import/reset.css";
+@import"../for-import/colors.css";
+
+.general-style-section {
+    margin-bottom: 2rem;
+}
+
+.btn {
+    padding: 10px;
+    width: 6rem; 
+    /* margin-right: 10px; */
```

Comment:
commented code can be removed throughout the file which will not add any value.

Reply:
removed